tf_encrypted/protocol/pond/triple_sources.py

```diff
@@ -104,6 +104,22 @@ def transpose_mask(self, a, perm):
 
     return a_t
 
+  def gather_mask(self, a, indices, axis):
```

Comment:
For posterity, can you include a mention of this somewhere, either in the commit history or in the top-level comment of this PR?

Reply:
Updated the top level PR :)